pkgs/os-specific/linux/jool/cli.nix

```diff
@@ -10,6 +12,10 @@ stdenv.mkDerivation {
 
   src = sourceAttrs.src;
 
+  patches = [
+    ./validate-config.patch
```

Comment:
What is the source of this patch? Did you write this yourself? do we want to upstream this=

Reply:
Yes. It would probably be useful upstream, but it's pretty rudimental as it is.
I will ask for feedback.

Reply:
Did that happen? Can you reference the communication with the upstream?

Reply:
Not yet, but I will be sure to link it here.